internal/httputil/httputil.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func BuildHTTPClient(cpw *CertPoolWatcher) (*http.Client, error) {
-	httpClient := &http.Client{Timeout: 10 * time.Second}
+	httpClient := &http.Client{Timeout: 2 * time.Minute}
```

Comment:
Now that we have the operatorhubio catalog installed by default, our e2e tests are entering and staying in exponential backoff because it takes longer for the operatorhubio catalog to be pulled, unpacked, and cached than the test catalog does.

Once we have #1028, we could consider excluding the operatorhubio catalog from resolution during these e2es. Even so, we likely want this value bumped up a bit to handle the default catalog without requests timing out.

Reply:
Can you add a TODO to revert this later?

Reply:
I reverted this. Seems to be a problem locally for me only.